modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java

```diff
@@ -366,6 +392,63 @@ private static Map<String, Object> _createCodeEditorElement(
 		).build();
 	}
 
+	private static void _includeRelatedObjectFields(
+			List<Map<String, Object>> codeEditorElements, Locale locale,
+			long objectDefinitionId2,
+			Predicate<ObjectField> objectFieldPredicate)
+		throws PortalException {
```

Comment:
I would name it `long objectDefinitionId`.

Reply:
I think it is clear because the method in the FOR specifies ObjectRelationshipLocalServiceUtil.getObjectRelationshipsByObjectDefinitionId2().

Reply:
@marcelabc Should I keep the `ObjectDefinition objectDefinition1` inside the for [here](https://github.com/liferay-bpm/liferay-portal/blob/8b60615c079f67d053dfb052314e5b817590b945/modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java#L410-L412)? Or should I rename it to `objectDefinition`?